### PR TITLE
Remove requirejs

### PR DESCRIPTION
**Pull request by sndrs**
## What does this change?

_this is a redux of #15845, which was maybe getting a bit hard to follow and had significant changes..._

- removes requirejs from frontend
- removes the webpack switch (we cannot turn off webpack)
- leaves `curl` for interactives

## What is the value of this and can you measure success?

one module bundler

## Does this affect other platforms - Amp, Apps, etc?

fingers crossed, no

## Tested in CODE?

- [x] tested on code

### CODE REVIEW

**Comment by regiskuckaertz** [comment]
giphy

File: static/src/javascripts-legacy/bootstraps/enhanced/media/video-player.js [hunk 1]
```diff
@@ -10,9 +10,7 @@ define([
     // In rjs this module is shimmed in the config
     // https://github.com/guardian/frontend/blob/webpack-redux/tools/__tasks__/compile/javascript/rjs.config.js#L42
     // We can move this up to the define() when we go 100% WP
```
Comment (SiAdcock) [inline static/src/javascripts-legacy/bootstraps/enhanced/media/video-player.js]: Can we action this comment now, or will there be a forthcoming PR to deal with comments like this?

Comment (sndrs) [inline static/src/javascripts-legacy/bootstraps/enhanced/media/video-player.js]: as discussed irl, we'll do this when cleaning out all the `#wp-rjs`s

**Review by gtrufitt — APPROVED** [review]
👍 💥
